src/gen/opcodes.rs

```diff
@@ -50,6 +51,12 @@ pub const ASSERT_BEFORE_HEIGHT_ABSOLUTE: ConditionOpcode = 87;
 // no-op condition
 pub const REMARK: ConditionOpcode = 1;
 
+// takes its cost as the first parameter, followed by future extensions
+// the cost is specified in increments of 10000, to keep the values smaller
+// This is a hard fork and is therefore only available when enabled by the
+// ENABLE_SOFTFORK_CONDITION flag
+pub const SOFTFORK: ConditionOpcode = 2;
```

Comment:
It's nice to have condition codes be large enough so they are in a different namespace than opcodes, which means a single dissassembler can work for programs representing clvm code and programs representing conditions without giving hints. Can we bump this up to, maybe say, 96 or 112?

Using `1` was sort of a mistake that maybe is was okay since `REM` and `q` are arguably kind of the same thing.

Reply:
sure, I'll change this to a larger value